stacks/spica/packages/material/resize/resize.directive.ts

```diff
@@ -91,13 +99,33 @@ export class MatResizeHeader implements AfterViewInit {
       this.setMatSortDisabled(false);
     }
     if (this._isDragging) {
-      const diff = event.pageX - this._startX;
-      if (diff != 0) {
-        this._width = this._startWidth + diff;
-        this.resize.emit(this._width);
-      }
+      if (!this.hasIntervalClass()) this.addIntervalRowClass();
+      this.setCellWidth(event);
+    }
+  }
+
+  setCellWidth(event) {
+    const diff = event.pageX - this._startX;
+    if (diff != 0) {
+      this._width = this._startWidth + diff;
+      this.resize.emit(this._width);
     }
   }
+  removeIntervalRowClass() {
+    this.renderer.removeClass(this.elemRow, this.elemRowClass);
+  }
+
+  addIntervalRowClass() {
+    this.renderer.listen("document", "mousemove", event => {
+      if (this.hasIntervalClass()) this.setCellWidth(event);
```

Comment:
Instead of if-check here, it will be better to remove the listener in removeIntervalRowClass. Otherwise, it cause triggering multiple closure functions